core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -289,6 +289,18 @@ public void iterateStatus(AlluxioURI path, final ListStatusPOptions options,
     });
   }
 
+  @Override
+  public void loadMetadata(AlluxioURI path, final ListStatusPOptions options)
+      throws FileDoesNotExistException, IOException, AlluxioException {
+    checkUri(path);
+    rpc(client -> {
+      ListStatusPOptions mergedOptions = FileSystemOptions.listStatusDefaults(
+          mFsContext.getPathConf(path)).toBuilder().mergeFrom(options).build();
```

Comment:
set the added flag explicitly here?

Reply:
@JySongWithZhangCe 

Better to add `.setLoadMetadataOnly(true)` explicitly here?

Reply:
Let's still keep the param as `ListStatusPOptions options`, to be consistent with other methods in this file

Reply:
Ok

Reply:
@JySongWithZhangCe 
we should add `.setLoadMetadataOnly(true)` explicitly here, right?
otherwise, how to ensure callers of this method has this set?

Reply:
I agree